src/params/progression.jl

```diff
@@ -98,6 +127,7 @@ end
 function resample!(
   rng::AbstractRNG,
   progressions, ages::AbstractArray{T} where T <: Real,
```

Comment:
```suggestion
  progressions, ages::AbstractVector{T} where T <: Real,
```
please add this change, we should allow only only 1-dimensional arrays here